daprdocs/content/en/reference/components-reference/supported-pubsub/setup-jetstream.md

```diff
@@ -97,6 +97,7 @@ spec:
 | rateLimit       |    N     | [Rate Limit]                               | `1024`                           |
 | hearbeat        |    N     | [Hearbeat]                                 | `10s`                            |
 | ackPolicy       |    N     | [Ack Policy]                               | `explicit`                       |
+| deliverPolicy   |    N     | One of: all, last, new, sequence, time     | `all`                            |
```

Comment:
```suggestion
| deliverPolicy   |    N     | One of: `all`, `last`, `new`, `sequence`, `time`     | `all`                            |
```

Reply:
@Jarema - Can you also remove the "deliverAll" from the Spec Metadata table. This line.
deliverAll | N | Set deliver all as Replay Policy | true

Reply:
Ah, sorry, missed that!
Removed.